streamparse/util.py

```diff
@@ -43,7 +43,9 @@ def activate_env(env_name=None):
     env.disable_known_hosts = True
     env.forward_agent = True
     env.use_ssh_config = True
-
+    # fix for config file load issue 
+    if env_config.get("ssh_password") != "":
```

Comment:
No need to be so specific here.  Just change it to `if env_config.get('ssh_password'):` and you'll be all set, since that'll skip `None` and empty string.